samples/net/sockets/echo_client/src/udp.c

```diff
@@ -252,7 +252,7 @@ void stop_udp(void)
 		k_delayed_work_cancel(&conf.ipv6.udp.recv);
```

Comment:
If you are sending a new version of this PR, you could tweak the commit subject a bit. Now it talks about echo_client but commit is for both echo-client and server. So the commit subject could be something like `samples: net: echo_client/server: Fix bug in socket close", also no need to put "." at the end of the subject line.